test/tbb/test_task_arena.cpp

```diff
@@ -1748,7 +1748,13 @@ struct enqueue_test_helper {
 
 //--------------------------------------------------//
 
-// This test should be first
+//! Test for uninitilized arena
+//! \brief \ref requirement \ref error_guessing
+TEST_CASE("Test current_thread_index") {
+    CHECK(tbb::this_task_arena::current_thread_index() < 0);
```

Comment:
```suggestion
    CHECK(tbb::this_task_arena::current_thread_index() == task_arena::not_initialized);
```

Reply:
I tried to do it but it ended with `/test/tbb/test_task_arena.cpp:1754: undefined reference to `tbb::detail::d1::task_arena_base::not_initialized'`